### DISCUSSION TITLE
ImGui on Android

### DISCUSSION THREAD

**Discussion by rh101** [axmolengine/axmol#906] [General]
I've managed to get ImGui working on the Android build of this engine, and while everything seems to be rendering correctly, the major issue I'm facing at the moment is how to get touch input and keyboard input to ImGui.

If anyone wants the Android code to mess with it themselves, then I can post it here in this discussion.

EDIT: I think I've found a way to get the touch (and possibly keyboard) input into ImGui, and if it works, I'll put up a PR with the changes.

**Comment by rh101** [comment]
The initial Android implementation should be working now, and there is a chance that the implementation may also work for iOS, but I don't have the time to test that out at the moment.

**Comment by rh101** [comment]
@halx99 This thread can be deleted, since I only created it to ask for some help with the issues related to getting ImGui for Android working, but that is all sorted now.

**Reply by halx99** [reply]
OK, let's lock it
